### ISSUE TITLE
LXQt 0.10, lx-monitor-config not restoring monitor preferences

### ISSUE THREAD

**Issue by lhorace** [lxde/lxqt#1101]
According to this bug report https://github.com/lxde/lxqt/issues/1096, this particular issue seems to already been fixed for 0.11 with https://github.com/lxde/lxqt-config/pull/95/commits/ab9543f24c2cac67d007c55f1fb37ae4a1e6c067. 

However, for the time being, is there any workaround one might try for 0.10?

Btw, there is no ~/.config/autostart/lxqt-config-monitor-autostart.desktop file on my box.

**Comment by pmattern** [comment]
As for #1096 `~/.config/autostart/lxqt-config-monitor-autostart.desktop` still gets written as usual, see second comment. So this is probably not what you're suffering from. Your issue is probably #878 which was affecting release 0.10 as well.
Both are fixed in master and will hence be fixed in release 0.11, too.

A workaround is an arbitrary desktop entry file `~/.config/autostart/somearbitraryname.desktop` invoking binary `xrandr` itself (which must be available of course).
E. g. something like this
```
[Desktop Entry]
Type=Application
Name=Screen setup by xrandr
Terminal=false
StartupNotify=false
TryExec=xrandr
Exec=xrandr --output Virtual-0 --pos 0x0 --mode 1024x768 --primary --output Virtual-1 --pos 1024x0 --mode 800x600
```
where values and names of outputs have of course to be adjusted to your needs, the latter can be seen by simply invoking `xrandr`.

That aside please always let us know which OS you're running and how LXQt was installed so that we can optionally try and reproduce your findings.

**Comment by dagelf** [comment]
Got the same problem on 1.1. Now trying 1.2.
